Makefile

```diff
@@ -1,15 +1,15 @@
 # Package configuration
 PROJECT = datasets
-COMMANDS = datasets/PublicGitArchive/borges-indexer/cmd/borges-indexer datasets/PublicGitArchive/multitool datasets/PublicGitArchive/pga
+COMMANDS = PublicGitArchive/borges-indexer/cmd/borges-indexer PublicGitArchive/multitool PublicGitArchive/pga
 
 # Including ci Makefile
 CI_REPOSITORY ?= https://github.com/src-d/ci.git
 CI_PATH ?= $(shell pwd)/.ci
 CI_VERSION ?= v1
-PKG_OS=linux darwin windows
+PKG_OS = linux darwin windows
```

Comment:
was that really the problem?

![(╯°□°）╯︵ ┻━┻)](https://media.giphy.com/media/2jlDkaipGfgGc/giphy.gif)